framework/src/kernels/MatReaction.C

```diff
@@ -20,6 +20,7 @@ MatReaction::validParams()
                        "kernel's nonlinear variable for v");
   params.addClassDescription("Kernel to add -L*v, where L=reaction rate, v=variable");
   params.addParam<MaterialPropertyName>("mob_name", "L", "The reaction rate used with the kernel");
```

Comment:
Do we know how many inputs rely on the default value? If there are none outside of the MOOSE repository, I would vote to follow @GiudGiud's suggestion and make this required and remove the default value

Reply:
Cool! It should be required. Almost like you need a deprecation for taking something from non-required to required.

I only see two instances where this would fail. Let me have this latest CIVET go through to make sure it all passes, then I'll make it a required and see how many things fail?